prolif/plotting/network/lignetwork.py

```diff
@@ -174,53 +174,12 @@ class LigNetwork:
         "HBDonor": 1,
         "XBDonor": 1,
     }
-    _JS_TEMPLATE = """
-        var ifp, legend, nodes, edges, legend_buttons;
-        function drawGraph(_id, nodes, edges, options) {
-            var container = document.getElementById(_id);
-            nodes = new vis.DataSet(nodes);
-            edges = new vis.DataSet(edges);
-            var data = {nodes: nodes, edges: edges};
-            var network = new vis.Network(container, data, options);
-            network.on("stabilizationIterationsDone", function () {
-                network.setOptions( { physics: false } );
-            });
-            return network;
-        }
-        nodes = %(nodes)s;
-        edges = %(edges)s;
-        ifp = drawGraph('%(div_id)s', nodes, edges, %(options)s);
-    """
-    _HTML_TEMPLATE = """
-        <html>
-        <head>
-        <script type="text/javascript" src="https://unpkg.com/vis-network@9.0.4/dist/vis-network.min.js"></script>
-        <link href="https://unpkg.com/vis-network@9.0.4/dist/dist/vis-network.min.css" rel="stylesheet" type="text/css" />
-        <style type="text/css">
-            body {
-                padding: 0;
-                margin: 0;
-                background: #fff;
-            }
-            .legend-btn.residues.disabled {
-                background: #b4b4b4 !important;
-                color: #555 !important;
-            }
-            .legend-btn.interactions.disabled {
-                border-color: #b4b4b4 !important;
-                color: #555 !important;
-            }
-        </style>
-        </head>
-        <body>
-        <div id="mynetwork"></div>
-        <div id="networklegend"></div>
-        <script type="text/javascript">
-            %(js)s
-        </script>
-        </body>
-        </html>
-    """  # noqa: E501
+
+    _JS_FILE = Path(__file__).parent / "network.js"
+    _HTML_FILE = Path(__file__).parent / "network.html"
+
+    _html_template = _HTML_FILE.read_text()
```

Comment:
Could you please keep the same name as before (uppercase)? It limits the chances of breaking changes for downstream users. Same for `_js_file_content` -> `_JS_TEMPLATE`